src/api/api-hooks.ts

```diff
@@ -30,3 +34,13 @@ export const usePromotedLessons = () => {
     const wixApi = useContext(WixAPIContext);
     return useSwr('promoted-lessons', wixApi.getPromotedLessons);
 };
+
+export const useUpcomingBookings = () => {
+    const wixApi = useContext(WixAPIContext);
```

Comment:
Should use `useWixApi` in every place there's `useContext(WixAPIContext)` for consistency.